paper/tex/results.tex

```diff
@@ -1,386 +1,146 @@
+\label{fig:pitqq}
+\end{figure*}
 
+The PIT histograms of \delight, \cmnn, \skynet, and \tpz\ feature an underrepresentation of extreme values, indicative of overly broad \pzpdf s, while the overrepresentation of extreme values for for \metaphor indicate overly narrow \pzpdf s.
+These five codes in particular have a free parameter for bandwidth, which may be responsible for this vulnerability, in spite of the opportunity for fine-tuning with perfect prior information.
+\flexzboost's ``sharpening'' parameter (described in Section~\ref{sec:flexzboost}) played a key role in diagonalizing the QQ plot, indicating a common avenue for improvement in the approaches that share this type of parameter.
+On the other hand, the three purely template-based codes, \bpz, \eazy, and \lephare, do not exhibit much systematic broadening or narrowing, which may indicate that complete template coverage effectively defends from these effects.
 
 \begin{table}
 \setlength{\tabcolsep}{2pt}
 \centering
-\caption{The fraction of ``catastrophic outlier'' PIT values.  We expect a value of 0.0002 for a proper Uniform distribution.  An excess over this small value indicates true redshifts that fall outside the non-zero support of the $p(z)$.  }\label{tab:pitoutlier}
+\caption{The catastrophic outlier rate as defined by extreme PIT values.
+We expect a value of 0.0002 for a proper Uniform distribution.
+An excess over this small value indicates true redshifts that fall outside the non-zero support of the $p(z)$.}
+\label{tab:pitoutlier}
 \begin{tabular}{lc}
 \hline
 \hline
- ``catastrophic outlier''\\ PIT fraction\\
+\Pz\ Code & fraction PIT$<10^{-4}$ or $>$0.9999\\
 \hline
-Photo-z Code & fraction PIT$<10^{-4}$ or $>$0.9999\\
+\annz       & 0.0265\\
+\bpz        & 0.0192\\
+\delight    & 0.0006\\
+\eazy       & 0.0154\\
+\flexzboost & 0.0202\\
+\gpz        & 0.0058\\
+\lephare    & 0.0486\\
+\metaphor   & 0.0229\\
+\cmnn       & 0.0034\\
+\skynet     & 0.0001\\
+\tpz        & 0.0130\\
 \hline
-\textsc{ANNz2} & 0.0265\\
-\textsc{BPZ} & 0.0192\\
-\textsc{Delight} & 0.0006\\
-\textsc{EAZY} & 0.0154\\
-\textsc{FlexZBoost} & 0.0202\\
-\textsc{GPz} & 0.0058\\
-\textsc{LePhare} & 0.0486\\
-\textsc{METAPhoR}& 0.0229\\
-\textsc{CMNN} & 0.0034\\
-\textsc{Skynet} & 0.0001\\
-\textsc{TPZ} & 0.0130\\
-\hline
-\textsc{TrainZ} & 0.0002\\
+\trainz     & 0.0002\\
 \end{tabular}
 \end{table}
 
+Though the spikes in the first and last bin of the PIT histogram were cut off in Figure~\ref{fig:pitqq} for visualization, the catastrophic outlier rates are provided in Table~\ref{tab:pitoutlier}.
```

Comment:
They are not cut off, if you blow up the figure you can see them.  I tried adding some padding to separate them from the axis markers, but apparently I didn't do a good enough job.  It's hard to see them because they are only 1-bin wide.

Reply:
I thought you actually cut out the highest and lowest [small fraction] of the probability range because the outliers at the ends screwed up the normalization between subplots.  How else did you get them to have the same y range?

Reply:
Oh, I read this as a horizontal cut, not a vertical cut.  Yes, they are all set to the same y-range, which does cut off the tops of the histogram.  The wording of "first and last bins" was what threw me, maybe re-word this a bit?